getting_started/topics/standalone/proc-logging-in-admin-console.adoc

```diff
@@ -23,7 +23,7 @@ image:images/admin-login.png[Admin console login screen]
 The initial screen for the admin console appears.
 +
 .Admin console
-image:images/admin-console.png[Admin console]
+image:{project_images}/admin-console.png[Admin Console]
```

Comment:
This image applies to the new Admin Console, but the version under rhsso-images is showing the same image.  It should be unchanged for rhsso-images.

Reply:
Updated this, but please double-check if this is correct now. I may have misunderstood the directions.